calico_cni/constants.py

```diff
@@ -70,3 +71,9 @@
 LOG_DIR = "/var/log/calico/cni"
 LOG_FORMAT = '%(asctime)s %(process)d [%(identity)s] %(levelname)s %(message)s'
 
+# Mesos namespaced data.
+MESOS_NS_KEY = "org.apache.mesos"
```

Comment:
Doesn't matter for this PR, but I should note it anyway.  

The `calico-cni` code made a mistake of using way too many constants when we should have just used inline string values.  Ultimately, making a new constant for something that's only used once or twice in the code just makes it harder to understand what the code is doing.

These are fine though because it's in line with what the rest of the code does.